### PR TITLE
V3: post code

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* add post code field to card component

### CODE REVIEW

File: Adyen/Components/Base/AddressInfo.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    /// Required for the US and Canada.
+    public var stateOrProvince: String?
+
+    /// The name of the street.
+    /// The house number should not be included in this field; it should be separately provided via houseNumberOrName.
+    public var street: String?
+
+    /// The name or code of apartment. Optional.
+    /// Will be included into houseNumberOrName.
+    public var apartment: String?
```
Comment (mohammedDehairy) [inline Adyen/Components/Base/AddressInfo.swift:55]: we don't need all those fields if its just postal code, right? actually we don't need a whole `AddressInfo` object, just the postal code as a string.

Comment (descorp) [reply Adyen/Components/Base/AddressInfo.swift:55]: we still need it for JSON mapping

File: Adyen/Assets/ko.lproj/Localizable.strings [hunk 2]
```diff
@@ -37,6 +37,7 @@
 "adyen.phoneNumber.placeholder" = "123–456–789";
 "adyen.card.cvcItem.placeholder.digits" = "%@자리";
 "adyen.emailItem.title" = "이메일 주소";
+"adyen.emailItem.placeHolder" = "이메일 주소";
```
Comment (mohammedDehairy) [inline Adyen/Assets/ko.lproj/Localizable.strings:40]: this line change is on some files but not others 🤔

Comment (descorp) [reply Adyen/Assets/ko.lproj/Localizable.strings]: 👍

**Review by kaphacius — APPROVED** [review]

**Pull request merged**